src/CMakeLists.txt

```diff
@@ -22,41 +21,46 @@ ExternalProject_Add(libint_library
     # download generalized to generating source by running libint_compiler
     DOWNLOAD_COMMAND ${STAGED_INSTALL_PREFIX}/bin/libint_compiler
     LOG_DOWNLOAD 1
-    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+    BINARY_DIR ${CMAKE_BINARY_DIR}/libint-src
+    CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${STAGED_INSTALL_PREFIX}
                -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
                -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+               -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
                -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+               -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
                -DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}
                -DCMAKE_INSTALL_INCLUDEDIR=${CMAKE_INSTALL_INCLUDEDIR}
                -DMAX_AM_ERI=${MAX_AM_ERI}
                -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS}
                -DBUILD_FPIC=${BUILD_FPIC}
                -DLIBC_INTERJECT=${LIBC_INTERJECT}
-    CMAKE_CACHE_ARGS -DCMAKE_C_FLAGS:STRING=${CMAKE_C_FLAGS}
-                     -DCMAKE_CXX_FLAGS:STRING=${CMAKE_CXX_FLAGS}
```

Comment:
Sure you want `CMAKE_<lang>_FLAGS` moved from `CMAKE_CACHE_ARGS` to `CMAKE_ARGS`? The former does a better job at preserving list variables.

Reply:
I haven't thought about that. I'll return `CMAKE_<lang>_FLAGS` to `CMAKE_CACHE_ARGS`